static/404.html

```diff
@@ -81,6 +81,26 @@ <h1 class="translatable">
 if(window.location.host.includes('demandetesdonnees.fr') || window.location.host.includes('1315')) lang = 'fr';
 if(window.location.host.includes('pedidodedados.org') || window.location.host.includes('1316')) lang = 'pt';
 
+// Adds 'lang' attribute to <html>
+let htmlSelector = document.querySelector('html');
```

Comment:
The region subtag is optional in the language tag (and it's not like we are targeting any specific regional language variant). Thus, we can do without the switch case. That's what I meant by a cleaner implementation. :) In fact, we can simplify the whole thing to one line:

```js
document.documentElement.lang = lang;
```